app/api-v1/routes/attachment/{id}.js

```diff
@@ -1,8 +1,26 @@
 // eslint-disable-next-line no-unused-vars
+const { BadRequestError } = require('../../../utils/errors')
 module.exports = function (attachmentService) {
   const doc = {
     GET: async function (req, res) {
-      res.status(500).json({ message: 'Not Implemented' })
+      const response = await attachmentService.getAttachmentByID(req.params.id)
+
+      if (req.headers.accept.includes('json') && response.filename === 'json') {
+        const json = JSON.parse(response.binary_blob)
+        res.status(200).json(json)
+      } else if (req.headers.accept.includes('json') && response.filename != 'json') {
+        throw new BadRequestError({ message: 'Filename does not match JSON' })
```

Comment:
I reckon saying something along lines of `not a JSON file` would be clearer to the user than talking about filename as renaming the file to `json` happens in code